.circleci/config.yml

```diff
@@ -216,6 +216,18 @@ jobs:
           name: Run danger-swift
           command: danger-swift ci
 
+  "Run Pod spec lint":
+    macos:
+      xcode: "11.2.0"
+    environment:
+      HOMEBREW_NO_AUTO_UPDATE: "1"
+    steps:
+      - checkout
+      - run:
+          name: Pod lib lint
+          command: |
+            pod lib lint --allow-warnings --verbose
```

Comment:
I've opened #104 to fix the warnings. I'll merge it as soon as the CI is passed, so we can remove the `--allow-warnings` flag.

Reply:
I think we should keep `--allow-warnings`. Lint also creates warnings when the Description is short than the summary. This would then lead to a failing CI job.